server/tests/auth.test.js

```diff
@@ -69,7 +69,7 @@ describe('User authentication Endpoints', () => {
       .send(falseUserEmail)
       .end((err, res) => {
         res.body.should.be.an('Object');
-        res.body.should.have.property('status').equal(400);
+        res.body.should.have.property('status').equal(550);
```

Comment:
What does the 550 status code mean to you?

Reply:
Means the account doesn't exist when the email used in not found

Reply:
That shouldn't be the correct status code for that.